benchmarks/instruction_counts/execution/runner.py

```diff
@@ -0,0 +1,240 @@
+import math
+import multiprocessing
+import subprocess
+import textwrap
+import threading
+import time
+from typing import Dict, List, Optional, Set, Tuple, Union
+
+from execution.work import PYTHON_CMD, SHELL, InProgress, WorkOrder
+from worker.main import WorkerFailure, WorkerOutput
+
+
+CPU_COUNT: int = multiprocessing.cpu_count()
+
+
+class WorkerFailed(Exception):
+    """Raised in the main process when a worker failure is detected."""
+    def __init__(self, wrapped_trace: Optional[str] = None) -> None:
+        self.wrapped_trace: Optional[str] = wrapped_trace
+        super().__init__()
+
+
+class CorePool:
```

Comment:
In TorchBench CI, we use the environment variable `GOMP_CPU_AFFINITY` and the `taskset` tool to set cpu affinity of the benchmark. It looks like the `CorePool` class sets cpu affinity similar to `taskset`, so I guess running the instruction count benchmark only requires setting `GOMP_CPU_AFFINITY` and `core_pool`?

Reply:
I think you have put much more thought into stabilizing the environment than I have. I added `GOMP_CPU_AFFINITY` to the launch command; thanks for the tip!